docs/src/cli/manage-stake-accounts.md

```diff
@@ -0,0 +1,74 @@
+# Manage Stake Accounts
+
+If you want to delegate stake to many different validators, you will need
+to create a separate stake account for each. If you follow the convention
+of creating the first stake account at seed "0", the second at "1", the
+third at "2", and so on, then the `solana-stake-accounts` tool will allow
+you to operate on all accounts with single innovations. You can use it to
+sum up the balances of all accounts, move accounts to a new wallet, or set
+new authorities.
+
+## Usage
+
+### Create a stake account
+
+Create and fund a derived stake account at the stake authority public key:
+
+```bash
+solana-stake-accounts new <SENDER_KEYPAIR> <BASE_KEYPAIR> <AMOUNT> \
```

Comment:
Does SENDER == FUNDING?  FUNDING_KEYPAIR makes more sense to me.  And BASE_KEYPAIR is the derivation root?

Reply:
"sender" is the counterpart to a "recipient" in `solana transfer`.   I'll think about it.  Mostly about where we want words with an "er" suffix or "ing".

Reply:
Went with FUNDING